iam-login-service/src/main/java/it/infn/mw/iam/core/oauth/profile/common/BaseIntrospectionHelper.java

```diff
@@ -108,6 +108,20 @@ protected void addAudience(Map<String, Object> introspectionResult,
 
   }
 
+  protected void addAcrClaimIfNeeded(OAuth2AccessTokenEntity accessToken,
+      Map<String, Object> introspectionResult) {
+
+    try {
+      Object acr = accessToken.getJwt().getJWTClaimsSet().getClaim("acr");
+      if (acr instanceof String acrString) {
+        introspectionResult.put("acr", acrString);
+      }
+    } catch (ParseException e) {
+      LOG.error("Error getting acr claim out of access token: {}", e.getMessage(), e);
+    }
+
+  }
```

Comment:
We're moving in a direction that the content of the access token will be printed claim by claim into the introspection endpoint but it something on me on my branch as soon as this PR will be merged and my branch rebased